kairon/shared/upload_handler/upload_handler_log_processor.py

```
@@ -0,0 +1,111 @@
+            ).filter(
+                Q(event_status__ne=EVENT_STATUS.COMPLETED.value) &
+                Q(event_status__ne=EVENT_STATUS.FAIL.value) &
+                Q(event_status__ne=EVENT_STATUS.ABORTED.value)
+            ).get()
+
+            if raise_exception:
+                raise AppException(
+                    f"Upload already in progress for collection: {collection_name}. Check logs."
+                )
+            in_progress = True
+        except DoesNotExist as e:
```

Ruff (0.12.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)